<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgMovement</class>
 <widget class="QDialog" name="DlgMovement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="2" column="8">
       <widget class="QLineEdit" name="leFilter">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="9">
       <widget class="QToolButton" name="btnSaveAndAccept">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../resources/res.qrc">
          <normaloff>:/storeinput.png</normaloff>:/storeinput.png</iconset>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLineEdit" name="leComment">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Մեկնաբանություն</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Բարկոդ</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="7">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>561</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="10">
       <widget class="QToolButton" name="btnSave">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../resources/res.qrc">
          <normaloff>:/save.png</normaloff>:/save.png</iconset>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Պահեստ</string>
        </property>
       </widget>
      </item>
      <item row="0" column="12">
       <widget class="QToolButton" name="btnExit">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../resources/res.qrc">
          <normaloff>:/logout.png</normaloff>:/logout.png</iconset>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="cbDstStore">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="leSearch">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="11">
       <widget class="QToolButton" name="btnMinimize">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../resources/res.qrc">
          <normaloff>:/minimize.png</normaloff>:/minimize.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>btnMinimize</zorder>
     <zorder>leComment</zorder>
     <zorder>label_2</zorder>
     <zorder>leSearch</zorder>
     <zorder>label_3</zorder>
     <zorder>btnExit</zorder>
     <zorder>cbDstStore</zorder>
     <zorder>label</zorder>
     <zorder>leFilter</zorder>
     <zorder>btnSave</zorder>
     <zorder>btnSaveAndAccept</zorder>
    </widget>
   </item>
   <item>
    <widget class="C5ClearTableWidget" name="tbl">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Անվանում</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Բարկոդ</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Առկա</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Քանակ</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Գին</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Goods code</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>X</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Քանակ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="C5LineEdit" name="leTotalQty">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Գումար</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="C5LineEdit" name="leTotalAmount">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>669</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>C5ClearTableWidget</class>
   <extends>QTableWidget</extends>
   <header location="global">c5cleartablewidget.h</header>
  </customwidget>
  <customwidget>
   <class>C5LineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">c5lineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>leSearch</tabstop>
  <tabstop>leComment</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/res.qrc"/>
  <include location="../resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
